crates/pallet-feeds/src/lib.rs

```diff
@@ -17,29 +18,78 @@ pub mod pallet {
     pub struct Pallet<T>(_);
 
     pub type PutDataObject = Vec<u8>;
+    pub type FeedId = u64;
+
+    // TODO: make it more generic
+    #[derive(Encode, Decode, Debug, Clone, Eq, PartialEq)]
+    pub struct ObjectMetadata {
+        pub feed_id: FeedId,
+        pub hash: H256,
+        pub number: u32,
```

Comment:
Is this the number of an object? Comments on public types especially would be very useful.

Also shouldn't metadata be just `Vec<u8>`?